tests/baselines/reference/tsbuild/resolveJsonModule/initial-build/include-of-json-along-with-other-include-and-file-name-matches-ts-file.js

```diff
@@ -0,0 +1,15 @@
+//// [/lib/initial-buildOutput.txt]
+/lib/tsc --b /src/tsconfig_withIncludeOfJson.json
+error TS5056: Cannot write file '/src/dist/src/index.d.ts' because it would be overwritten by multiple input files.
+exitCode:: ExitStatus.DiagnosticsPresent_OutputsSkipped
```

Comment:
does this match the intent of the test? I can't tell from the test name.

Reply:
this is with config  file containing include pattern for json explicitly along with other tsc file include. And json name (index.json) is same as ts file (index.ts)